qrenderdoc/Windows/TextureViewer.cpp

```diff
@@ -741,7 +846,21 @@ TextureViewer::TextureViewer(ICaptureContext &ctx, QWidget *parent)
 
   ui->textureList->setModel(new TextureListItemModel(m_Ctx, this));
   ui->textureList->setItemDelegate(new TextureListItemDelegate(ui->textureList));
+  ui->textureList->viewport()->installEventFilter(this);
```

Comment:
Why is this event filter installed here? TextureViewer doesn't implement `eventFilter` so this seems like it wouldn't do anything.